docs/options.md

```diff
@@ -196,6 +196,7 @@ Valid options:
 
 - `"babel"` (via [@babel/parser](https://github.com/babel/babel/tree/master/packages/babel-parser)) _Named `"babylon"` until v1.16.0_
 - `"babel-flow"` (Same as `"babel"` but enables Flow parsing explicitly to avoid ambiguity) _First available in v1.16.0_
+- `"babel-ts"` (Like as `"babel"` but enables TypeScript parsing which diffs from Flow) _First available in v1.19.0_
```

Comment:
```suggestion
- `"babel-ts"` (Similar to `"babel"` but uses Babel’s TypeScript plugin to parse type annotations instead of usig Flow) _First available in v1.19.0_
```

Perhaps? Maybe something similar but better worded?

Reply:
Thanks, I am not a native English speaker. 😂